src/pages/iou/request/step/IOURequestStepScan/index.native.tsx

```diff
@@ -237,15 +295,14 @@ function IOURequestStepScan({
                 const source = `file://${photo.path}`;
                 IOU.setMoneyRequestReceipt(transactionID, source, photo.path, action !== CONST.IOU.ACTION.EDIT);
 
-                if (action === CONST.IOU.ACTION.EDIT) {
-                    FileUtils.readFileAsync(source, photo.path, (file) => {
+                FileUtils.readFileAsync(source, photo.path, (file) => {
```

Comment:
Did you add this or is this coming from resolving conflicts?

Reply:
Conflicts. Taking a look to make sure I'm not breaking anything there

Reply:
Nice catch. This did get messed up during conflict resolution. Thanks!

Reply:
Ah, I was wrong. I changed this to have the file available